src/components/AddActivityModal/AddActivityModal.js

```diff
@@ -5,22 +5,24 @@ import { useNavigate } from "react-router-dom";
 
 
 
-
-const AddActivityModal = ({createActivity}) => {
+const AddActivityModal = ({createActivity, activities, currentUser}) => {
+  // console.log(currentUser?.id);
   const [newActivity, setNewActivity] = useState({
     category:"",
-    activity:"",
+    activity_name:"",
+    activity_photo:"",
     start_time:"",
-    duration:"",
+    duration: 2.0,
```

Comment:
Is this supposed to be defaulted to 2?

Reply:
For now it is. The input field is making it a string and we need it to be a float per our rails models (and my filtering logic I hope to implement before monday).  Just need to make our input field only take numbers.